cc_licenses/templates/includes/breadcrumbs.html

```diff
@@ -3,10 +3,11 @@
   <ul>
       {% block breadcrumbs-li %}
       <li><a href="#">{% trans "Home" %}</a></li>
-      <li><a href="#">{% trans "Child page" %}</a></li>
-      <li><a href="#">{% trans "Child child page" %}</a></li>
-      {% if legalcode %}
-        <li class="is-active"><a href="#" aria-current="page displayed">{% include "includes/snippet/license_medium_title.html" %}</a></li>
+      <li><a href="#">{% trans "License" %}</a></li>
+      {% if 'legalcode' in request.path %}
+        <li class="is-active"><a href="#" aria-current="page displayed">{% trans "Legal Code for" %} {% include "includes/snippet/license_medium_title.html" %}</a></li>
+      {% else %}
+      <li class="is-active"><a href="#" aria-current="page displayed">{% trans "License Deed for" %} {% include "includes/snippet/license_medium_title.html" %}</a></li>
```

Comment:
It looks like this will display "license deed for" on every page that isn't a license page. (breadcrumbs.html is included unconditionally in base.html.)  Which isn't a big deal right now since we only have those two kinds of pages. We could just put the right breadcrumbs in deed_40.html and legalcode_40_page.html. Or let it go for now. What do you think?

Reply:
Thanks, I think we should do what you suggest since if/when we add another page we won't have to hunt down why that page is displaying `License Deed For`.

Reply:
pushed changes